pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -94,25 +96,24 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     atpublic
     bidict
-    filelock
     multipledispatch
     numpy
     pandas
     parsy
-    pooch
     pyarrow
+    pyarrow-hotfix
     python-dateutil
     pytz
     rich
     sqlglot
     toolz
     typing-extensions
-  ]
-  ++ pooch.optional-dependencies.progress
-  ++ pooch.optional-dependencies.xxhash;
+  ];
 
   nativeCheckInputs = [
     pytestCheckHook
+    black
```

Comment:
Can we remove tests that depend on black?

Reply:
Why? It seems easier to just depend on black.

Reply:
In https://github.com/NixOS/nixpkgs/pull/274092#issuecomment-1858157156, @SuperSandro2000 said, 
> It reduces the amount of rebuilds triggered and by that potential breakages when bumping linters. It is nice that people want to keep high quality code but if we update e.g. black and a new rule breaks many things that is just plain annoying and unnecessary. Hence if the linters are not touching our code, the rule of thumb is to always remove them to keep maintenance required lower.

I agree with this opinion.

Reply:
We don't test code quality here. Black is an optional dependency.